platform/ios/scripts/package.sh

```diff
@@ -207,17 +222,18 @@ SEM_VERSION=$( git describe --tags --match=ios-v*.*.* --abbrev=0 | sed 's/^ios-v
 SHORT_VERSION=${SEM_VERSION%-*}
 if [[ ${BUNDLE_RESOURCES} ]]; then
     cp -pv LICENSE.md "${OUTPUT}/static/${NAME}.framework"
-    cp -rv platform/ios/app/Settings.bundle "${OUTPUT}/static/${NAME}.framework"
+    cp -rv platform/ios/app/Settings.bundle ${STATIC_SETTINGS_DIRECTORY}
```

Comment:
We need to rethink `$BUNDLE_RESOURCES`. I added this variable to support `make ifabric`, which outputs an all-in-one static framework and nothing else. But now `make ipackage` needs to simultaneously output a static framework, a dynamic framework, and some auxiliary resources like Settings.bundle.

Before this change, `make ipackage` would place the LICENSE.md and Settings.bundle directly under pkg/, so that users of the dynamic framework can also access those resources easily. Now they’re hidden inside the static framework.